.gitignore

```diff
@@ -10,3 +10,4 @@ out
 tsconfig.tsbuildinfo
 *.vsix
 *.virtual.ts
+auto-imports.d.ts
```

Comment:
`auto-imports.d.ts` is useful file in a Web application project.

Is this change as your expected? I think this file is not supposed to be added into `.gitignore`

Reply:
<img width="1329" alt="image" src="https://github.com/user-attachments/assets/6936413f-840a-483e-a8dc-a0246eb6b00d" />

Yes, this is expected, and it does play a large part in the type prompt. However, if the automatic import of `composables` is enabled, the file may conflict due to different devices running at different development times (for example, windows uses `\` as a separator). Of course, this file will be automatically generated/updated every time the service is started, so in order to avoid polluting the git log, I listed it in `. gitignore.

Reply:
Because my personal computer is a macos system and my work computer is a windows system, I feel that I suffer greatly from this problem. 🤮

Reply:
OK, sounds it does make sense ! 😄